orchestrator/main_loops.go

```diff
@@ -275,17 +278,38 @@ func (p *gravityOrchestrator) EthSignerMainLoop(ctx context.Context) (err error)
 func (p *gravityOrchestrator) BatchRequesterLoop(ctx context.Context) (err error) {
 	logger := p.logger.With().Str("loop", "BatchRequesterLoop").Logger()
 
+	var estimatedGasCosts = []int64{
```

Comment:
1. Can we move this outside the function?
2. Can we also add a small godoc explaining what this is, how it's used, etc...?

Reply:
Done ✅